### PR TITLE
Implement ClipboardText on IKeyboard

### PR DESCRIPTION
**Pull request by Beyley**
# Summary of the PR
Add the ability to change the system clipboard

# Related issues, Discord discussions, or proposals
#732

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/Silk.NET?pullRequest=733) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/Silk.NET?pullRequest=733) before we can accept your contribution.<br/><br/>:x: Beyley [sign now](https://cla.dotnetfoundation.org/dotnet/Silk.NET?pullRequest=733)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/Silk.NET?pullRequest=733) it.</sub>

File: src/Input/Silk.NET.Input.Common/Interfaces/IKeyboard.cs [hunk 1]
```diff
@@ -15,6 +15,11 @@ public interface IKeyboard : IInputDevice
         /// The keys this keyboard has available.
         /// </summary>
         IReadOnlyList<Key> SupportedKeys { get; }
+        
+        /// <summary>
+        /// The text currently in the clipboard
```
Comment (ThomasMiz) [inline src/Input/Silk.NET.Input.Common/Interfaces/IKeyboard.cs:20]: XML Comments should end on a period:

```suggestion
        /// The text currently in the clipboard.
```

**Comment by ThomasMiz** [comment]
Looks good to me! Did you do some basic testing to make sure it's working correctly on both SDL and GLFW?

**Comment by Beyley** [comment]
> 

My computer ran out of ram and crashed when i tried to compile, i asked a friend to compile the project while they went out to the mall but i havent heard back yet, if someone else can test it i would be super grateful

**Comment by ThomasMiz** [comment]
I feel ya. VS2022 was completely unusable for me until I upgraded from 8 to 16GBs of RAM.

I did some testing and found [this difference in behaviour](https://github.com/dotnet/Silk.NET/issues/732#issuecomment-1001693407) between the backends, but there's not much we can do about that. That aside, it's working as intended on my system.

**Review by ThomasMiz — APPROVED** [review]

**Pull request merged**